engine/examples/running_ssh_service.md

```diff
@@ -13,7 +13,7 @@ quick access to a test container.
 ```Dockerfile
 FROM ubuntu:16.04
 
-RUN apt-get update && apt-get install -y openssh-server
+RUN apt-get update && apt-get install -y openssh-server --force-yes
```

Comment:
Can you explain why this is needed? I tried this example, and it works fine without this option

Reply:
Yes, this is the same as the `-y` already provided.

Reply:
Sorry that, I used the different base image, I copy the code, I missed some dependencies in that image, so I need to add `--force-yes` to confirm the installation of the dependencies.